app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] The `user` needs` to be the author with class_name user so the database will render the author of the comment.